### DISCUSSION TITLE
To provision a AWS NLB to be reconciled by AWS LBC controller(instead of in-tree legacy controller) using Gateway resource

### DISCUSSION THREAD

**Discussion by mithunkrb** [Kong/kubernetes-ingress-controller#5818] [General]
With service as a Load Balancer, there are annotation to be used to deploy a AWS NLB (reconciled by latest AWS LBC https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.7/guide/service/annotations/#lb-type) which is recommended way by AWS to deploy a Load Balancer. 
Example annotations:
      service.beta.kubernetes.io/aws-load-balancer-attributes: load_balancing.cross_zone.enabled=true
      service.beta.kubernetes.io/aws-load-balancer-scheme: "internet-facing"
      service.beta.kubernetes.io/aws-load-balancer-type: "external"

However, when using Gateway resource in Kong with KIC, same annotations has no effect instead its deploying classic load balancer itself. **Could you please help with the parameters or annotations to be used with Gateway resource when using Kong with KIC to deploy AWS NLB** to be reconciled by latest AWS LBC ?

Note: Classic load balancer which is reconciled by in-tree legacy controller is not recommended by AWS anymore as it has become previous generation LB.

**Comment by AfrazHussain** [comment]
Facing the same issue. Did you ever find a solution to this?

**Reply by mithunkrb** [reply]
Not yet, however I found this **Draft/Open** PR related to this issue in aws-load-balancer controller repo: 
https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/3146.
